src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1157,7 +1157,8 @@ static bool processContextAttributeVector (ContextElement*
                                            double&                                    coordLat,
```

Comment:
I think we should add an entry about this change in the CHANGES_NEXT_RELEASE file. Although this hack is temporal and hopefully it will be fixed before releasing 0.19.0, it is not completely sure that we achieve that, so it is better to have the "mark" in the CHANGES_NEXT_RELEASE just in case (if we implement a definitive fix for #716 before closing 0.19.0, that entry will be adjusted)